docs/usage/administration/admin_api/federation.md

```diff
@@ -74,11 +74,11 @@ The following fields are returned in the JSON response body:
   Destinations objects contain the following fields:
   - `destination` - string - Name of the remote server to federate.
   - `retry_last_ts` - integer - The last time Synapse tried and failed to reach the
-    remote server, in ms. This is `0` if no error has occurred.
+    remote server, in ms. This is `0` if no further retrying occuring.
   - `retry_interval` - integer - How long since the last time Synapse tried to reach
-    the remote server before trying again, in ms.
+    the remote server before trying again, in ms. This is `0` if no further retrying occuring.
   - `failure_ts` - integer - The first time Synapse tried and failed to reach the
-    remote server, in ms. This is `0` if no error has occurred.
+    remote server, in ms. This is `null` if no error has occurred.
```

Comment:
```suggestion
  - `failure_ts` - optional integer - The first time Synapse tried and failed to reach the
    remote server, in ms. This is `null` if communication with the remote server has never failed.
```

Reply:
Actually, maybe it's better to say "nullable integer", because the field will always be present? Or maybe we just say "integer" elsewhere in the docs?

Reply:
This is the content of the database:
```
select * from destinations;
 destination | retry_last_ts | retry_interval | failure_ts | last_successful_stream_ordering
-------------+---------------+----------------+------------+---------------------------------
 matrix.org  |             0 |              0 |            |                           29083
```

The result should be:
```json
      {
         "destination": "matrix.org",
         "retry_last_ts": 0,
         "retry_interval": 0,
         "failure_ts": null,
         "last_successful_stream_ordering": 1234
      }
```